docs/kafka-catalog.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Kafka catalog"
+slug: /kafka-catalog
+date: 2024-4-22
+keyword: kafka catalog
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
+Kafka catalog is a messaging catalog that offers the ability to manage Kafka topics metadata.
```

Comment:
You'd better change to Apache Kafka for the first time you mentioned about Kafka.